examples/visualization_demo/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+project(ignition-rendering-visualization-demo)
+
+find_package(ignition-rendering6)
+
+if (APPLE OR UNIX)
+  find_package(GLUT REQUIRED)
+  include_directories(SYSTEM ${GLUT_INCLUDE_DIRS})
+  link_directories(${GLUT_LIBRARY_DIRS})
+
+  find_package(OpenGL REQUIRED)
+  include_directories(SYSTEM ${OpenGL_INCLUDE_DIRS})
+  link_directories(${OpenGL_LIBRARY_DIRS})
+
+  set(TARGET_THIRD_PARTY_DEPENDS
+		${TARGET_THIRD_PARTY_DEPENDS}
+		${OPENGL_LIBRARIES}
+		${GLUT_LIBRARIES}
+  )
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations")
+endif()
+
+if (NOT APPLE)
+  find_package(GLEW REQUIRED)
+  set(TARGET_THIRD_PARTY_DEPENDS
+    ${TARGET_THIRD_PARTY_DEPENDS}
+    GLEW::glew)
```

Comment:
I was not able to build the demo with these lines as it complains about glew

I had to replace it with:

```
  if (WIN32)
    set(TARGET_THIRD_PARTY_DEPENDS
      ${TARGET_THIRD_PARTY_DEPENDS}
      GLEW::glew
    )
  else ()
    set(TARGET_THIRD_PARTY_DEPENDS
      ${TARGET_THIRD_PARTY_DEPENDS}
      GLEW
    )
  endif()
```

Reply:
FIxed in [`ca3ac18`](https://github.com/ignitionrobotics/ign-rendering/pull/326/commits/ca3ac18975bb64431ed2b581d648d6815b86cfb0)